pinot-broker/src/main/java/org/apache/pinot/broker/cursors/FsResponseStore.java

```diff
@@ -238,7 +238,7 @@ protected long writeResultTable(String requestId, ResultTable resultTable)
 
     try {
       pinotFS.copyFromLocalFile(tempResultTableFile.toFile(), dataFile);
-      return pinotFS.length(dataFile);
+      return tempResultTableFile.toFile().length();
```

Comment:
nit: each call to `toFile` is allocating a new `File` object - can be consolidated into a single call and reused.

Reply:
Fixed